---
- hosts: ip0*
  tasks:
    - name: Attempt and gracefull roll back demo
      block:
        - debug: msg='I execute normally'
          notify: run me even after an error
        - command: /bin/false
      rescue:
        - debug: msg= 'make sure all handlers run'
          meta: flush_handlers
  handlers:
    - name: run me even after an error
      debug: 
        msg: '실행되어야 하는데.. 왜 안되지?'